src/Commands/GenerateZipCommand.php

```diff
@@ -25,24 +29,26 @@ class GenerateZipCommand extends Command
     /**
      * Execute the console command.
      *
+     * @param CloudinaryEngine $engine
+     *
      * @return void
      */
     public function handle(CloudinaryEngine $engine)
     {
-
-        if(! config('cloudinary.cloud_url')) {
+        // [CN] this code duplicates GenerateArchiveCommand, consider refactoring
+        if (! config('cloudinary.cloud_url')) {
```

Comment:
It should simply be one Generate Zip/Archive class. Will delete one of the files here